### ISSUE TITLE
bun wiptest - error: Could not resolve: "cache-manager"

### ISSUE THREAD

**Issue by victorhnogueira** [oven-sh/bun#279]
I'm trying to run `bun wiptest` in my newly created [nest.js](https://github.com/nestjs/nest) project tha uses Jest and I'm getting the following Error message:
```bash
$ bun wiptest

app.controller.spec.ts:


error: Could not resolve: "cache-manager". Maybe you need to "bun install"?

            const cacheManager = (0, load_package_util_1.loadPackage)('cache-manager', 'CacheModule', () => require('cache-manager'));
```
The message suggest to run `bun install` but I had already did that and didn't work.
I'm running on WSL2 with Ubuntu 20.04 and the kernel is updated.

```bash
$ uname -a
Linux XXX 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```
```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04 LTS
Release:        20.04
Codename:       focal
```

**Comment by marvreichmann** [comment]
i'm having the exact same issue with both NestJS 8.x and 9.x. I'm on MacOS 12.4. 
Bun is on v0.1.5

**Comment by calebeaires** [comment]
Same here. Tried to run it on a builded Nestjs app (v8)

**Comment by birkskyum** [comment]
Things have moved. Getting this now:

```sh
➜ bun test
bun test v0.8.2 (f24ca390)

src/app.controller.spec.ts:
 5 | export class AppController {
 6 |   constructor(private readonly appService: AppService) {}
 7 | 
 8 |   @Get()
 9 |   getHello(): string {
10 |     return this.appService.getHello();
               ^
TypeError: undefined is not an object (evaluating 'this.appService.getHello')
      at getHello (/Users/admin/repos/nest-test/test2/src/app.controller.ts:10:11)
      at /Users/admin/repos/nest-test/test2/src/app.controller.spec.ts:19:13
✗ AppController > root > should return "Hello World!" [0.15ms]

 0 pass
 1 fail
Ran 1 tests across 1 files. [123.00ms]
```


Also there is a tracking issue here:
- #1641

**Comment by dylan-conway** [comment]
As a workaround, you can add the dependencies to `AppController` to avoid this error
```diff
- import { Controller, Get } from '@nestjs/common';
+ import { Controller, Get, Dependencies } from '@nestjs/common';
import { AppService } from './app.service';

@Controller()
+ @Dependencies(AppService)
export class AppController {
```
This will be solved when bun supports decorator metadata.

**Comment by Electroid** [comment]
This is fixed as of Bun v1.0.7. If are still running into this bug, please feel free to re-open the issue.

```
git clone git@github.com:nestjs/nest.git --depth=1
cd nest/sample/26-queues/
npm i --lockfile-version 3 --frozen-lockfile
```

See #6682, for why `npm i` is required here. We will fix that.

```
❯ bun install
bun install v1.0.7 (fa1bbec0)
[11.37ms] migrated lockfile from package-lock.json
 + @nestjs/bull@10.0.1
 + @nestjs/common@10.2.6
 + @nestjs/core@10.2.6
 + @nestjs/platform-express@10.2.6
 + bull@4.10.4
 + dotenv@16.3.1
 + reflect-metadata@0.1.13
 + rimraf@5.0.5
 + rxjs@7.8.1
 + @nestjs/cli@10.1.18
 + @nestjs/schematics@10.0.2
 + @nestjs/testing@10.2.6
 + @types/bull@3.15.9
 + @types/dotenv@8.2.0
 + @types/express@4.17.18 (v4.17.20 available)
 + @types/jest@29.5.6
 + @types/node@20.8.7
 + @types/supertest@2.0.15
 + @typescript-eslint/eslint-plugin@5.62.0 (v6.7.4 available)
 + @typescript-eslint/parser@5.62.0 (v6.9.0 available)
 + eslint@8.42.0 (v8.52.0 available)
 + eslint-config-prettier@9.0.0
 + eslint-plugin-import@2.29.0
 + jest@29.7.0
 + prettier@3.0.3
 + supertest@6.3.3
 + ts-jest@29.1.1
 + ts-loader@9.4.4
 + ts-node@10.9.1
 + tsconfig-paths@4.2.0
 + typescript@5.2.2

 810 packages installed [881.00ms]
```

```
❯ bun test
bun test v1.0.7 (fa1bbec0)

src/app.controller.spec.ts:
✓ AppController > root > should return "Hello World!" [0.33ms]

 1 pass
 0 fail
 2 expect() calls
Ran 1 tests across 1 files. [215.00ms]
```
